### PR TITLE
Adds Terraform to stand up an ECS Service

### PR DESCRIPTION
**Pull request by chtakahashi**
Adds the terraform required to provision an ECS Service

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -30,3 +32,70 @@ variable "scan_on_push" {
...
+variable "ecs_subnet_ids" {
+  description = "Subnet IDs for the ECS tasks."
+  type        = list(string)
+}
+
+variable "repo_tag" {
+  type        = string
+  description = "The tag to identify and pull the image in ECR repo"
+  default     = "latest"
+}
```
Comment (LinuxBozo) [inline variables.tf:67]: These variables that are prefixed with `repo` need cleaned up a bit to provide more context. Which belong to ECR, and which to GitHub?

File: variables.tf [hunk 2]
```diff
@@ -30,3 +32,70 @@ variable "scan_on_push" {
...
+variable "repo_owner" {
+  description = "the name of the repo owner"
+  type        = string
+  default     = "CMSgov"
+}
+
+variable "repo_name" {
+  description = "the name of the repository"
+  type        = string
+}
```
Comment (LinuxBozo) [inline variables.tf:101]: Again, more `repo` shenanigans.

File: variables.tf [hunk 3]
```diff
@@ -30,3 +32,70 @@ variable "scan_on_push" {
...
+
+variable "environment" {
+  type        = string
+  description = "Environment name"
+}
+
+variable "task_name" {
+  type        = string
+  description = "Name of the task to be run"
+}
```
Comment (LinuxBozo) [inline variables.tf:51]: Do we really need to pass all this in, or can we do it with just `environment`?  Or can we use other variables we're already passing, like github repo name?

File: outputs.tf [hunk 4]
```diff
@@ -1,9 +1,4 @@
-output "ecr_arn" {
-  description = "Full ARN of the repository."
-  value       = aws_ecr_repository.main.arn
-}
-
-output "ecr_repo_url" {
-  description = "The URL for the image created."
-  value       = aws_ecr_repository.main.repository_url
+output "security_group_id" {
```
Comment (LinuxBozo) [inline outputs.tf:1]: Why is the only output a security group id?

**Review by LinuxBozo — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (chtakahashi) [reply variables.tf:67]: updating to ecr_repo_tag

File: variables.tf (hunk 2 above)
Comment (chtakahashi) [reply variables.tf:101]: updating to github_repo for both

**Review by dynamike — APPROVED** [review]

File: outputs.tf (hunk 4 above)
Comment (chtakahashi) [reply outputs.tf]: outdated, removing this

File: variables.tf (hunk 3 above)
Comment (chtakahashi) [reply variables.tf]: removed app_name and task_name, filling in with just github_repo_name

**Pull request merged**